src/Nri/Ui/SideNav/V2.elm

```diff
@@ -240,9 +240,8 @@ viewLockedEntry extraStyles entryConfig =
 
 sharedEntryStyles : List Style
 sharedEntryStyles =
-    [ paddingLeft (px 20)
-    , paddingRight (px 20)
-    , height (px 45)
+    [ padding2 (px 13) (px 20)
+    , Css.property "word-break" "break-word"
```

Comment:
Looks like MDN says `word-break: break-word` is deprecated, how do you feel about using the alternate suggestions?

<img width="759" alt="image" src="https://user-images.githubusercontent.com/32229300/158450326-123ddb1f-f440-4eaf-ba0a-192e82a40aa5.png">

https://developer.mozilla.org/en-US/docs/Web/CSS/word-break#values

Reply:
Great catch, thank you!!! 🙏